### ISSUE TITLE
Conflict between SE model training and publish

### ISSUE THREAD

**Issue by ChenYouyuan2020** [ArcInstitute/state#202]
1. Share gene numbers : 
In publish, it is  256 (256 genes randomly sampled from the full gene set, This results in 1,280 (512+512+256)  genes for which expression values must be predicted per cell ). 
But SE-600M/config.yaml -> dataset.S = 512, should be 256?

2. Loss function: 
In publish, L_gene and L_cell both used MSE loss, but in config TabularLoss was be used. Which loss function  have better performance when we train SE Model?

3. cell_sentence_counts 
in src/emb/data/loader.py, line 574:  
cell_sentence_counts[c, :] = 100 * expression_weights[c, cell_sentences[c, :].to(torch.long)] 
Batch_sentence_counts is raw count data (after log1p) should not mean is gene dims?
it seems to be cell_sentence_counts and counts( which be used as batc h_weight) should change their location. 
Such as in in src/emb/data/loader.py, line 387:

(bs, xx, yy, batch_weight, mask, cell_total_counts, cell_sentence_counts) = self.sample_cell_sentences(
                counts, dataset, shared_genes, valid_mask, downsample_fraction
            ) 

---------------------------->change to --------------------->

(bs, xx, yy,cell_sentence_counts, mask, cell_total_counts, batch_weight) = 
self.sample_cell_sentences(
                counts, dataset, shared_genes, valid_mask, downsample_fraction
            ) 

4. I am not sure which choose is right and thanks for your excellent work!

**Comment by abhinadduri** [comment]
hi, thanks for making an issue!

1. the correct number that was used in training is 256, though 512 also performs similarly
2. tabular loss is more stable and performs better
3. I'm not quite sure I follow the question, could you explain further?
